files/es/learn/html/introduction_to_html/advanced_text_formatting/index.md

```diff
@@ -336,7 +336,7 @@ Las citas se escriben en cursiva de forma predeterminada.
 Las fuentes de citación que necesitas son:
 
 - `http://www.brainyquote.com/quotes/authors/c/confucius.html` para la citas de Confucio
-- `http://www.affirmationsforpositivethinking.com` para "La necesidad de eliminar el diálogo interno negativo".
+- `http://example.com/affirmationsforpositivethinking` para "La necesidad de eliminar el diálogo interno negativo".
```

Comment:
Should we update them to https?

```suggestion
- `https://example.com/affirmationsforpositivethinking` para "La necesidad de eliminar el diálogo interno negativo".
```

Reply:
Actually, I also wanted to replace it with `https`, but since in `en-US` version use `http` (https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Advanced_text_formatting), I decided to leave it like that.

So I think you can do it as you want here :)